### ISSUE TITLE
[Bug Report][3.6.4] Accessibility: VCheckbox using the hide-details prop adds an 'aria-describedby' attribute referencing a non-existent element

### ISSUE THREAD

**Issue by ldavisCEI** [vuetifyjs/vuetify#19794]
### Environment
**Vuetify Version:** 3.6.4
**Vue Version:** 3.4.27
**Browsers:** Edge 124.0.0.0
**OS:** Windows 10

### Steps to reproduce
Use the `hide-details` (or `hide-details="auto"`) prop on a VCheckbox component.

### Expected Behavior
The `aria-describedby` attribute should not be present, or should be empty or null.

### Actual Behavior
The `aria-describedby` attribute refers to a messages element that doesn't exist, causing accessibility check errors.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqNj8EKwjAQRH8l5GztXWLB7zAetsmWFtdNaJIiiP9uTASrePCy7D5mh5njTYbZtAfvt0tCuZMq4sUTROw0C6GWBrwvazmM4wgT4/xCFY5ozr27CoIeaa/lOFlsLGYlBTEABdRSrGGn2vfb/1aQovtyypJKf1kW9JH4SWoh1a6KJrY4ZJHVLO8bTkRlnB6cKGFi)

**Comment by sky0matic** [comment]
Seems to be the case with any inputs from Vuetify
